fig2_validation.py

```diff
@@ -28,15 +28,15 @@
 h.load_file('import3d.hoc')
 
 
- 
-    
+

```

Comment:
Can you provide this change without whitespace changes? These edits hide the substantive changes... or if you want the whitespace edits, can you make that in a separate commit?

Reply:
Done, preparing new commit